app/assets/javascripts/study-overview.js

```diff
@@ -0,0 +1,132 @@
+/**
+* @fileoverview Default view of Explore tab in Study Overview
+*
+* Shows "Clusters" and sometimes "Genomes", etc.
+*/
+
+const study = window.SCP.study
+
+if (typeof window.SCP.pushedPendingPageViewEvent === 'undefined') {
+  window.SCP.startPendingEvent('user-action:page:view:site-study',
+    { speciesList: window.SCP.taxons },
+    'plot:',
+    true)
+
+  // Flag to ensure we only log this pending event once per page view
+  window.SCP.pushedPendingPageViewEvent = true
```

Comment:
let's not use global variables for things like this.  Maybe we can create a parameter "onceOnly" on startPendingEvent (and maybe also on "log") and then metrics-api can can keep a module-scoped list to handle making sure that events like this only fire/start once.  (And I realize the global wasn't added on this change, but now is a good time to get rid of it)

Reply:
[Done](23f9eb0).  The global made sense initially, but it seems one of our refactors has made `window.SCP.pushedPendingPageViewEvent` obsolete.  So I simply removed it, and manually verified it only logs once.